stingray/pulse/tests/test_modeling.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from stingray.pulse.modeling import fit_sinc, fit_gaussian
+
+np.random.seed(0)
+
```

Comment:
Anyway, I added two more doctests for `sinc_square_deriv` and `sinc_square_model` :)